src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -439,6 +439,17 @@ public static bool IsTargetReadyForAeWithKeyStore()
                 ;
         }
 
+        public static bool IsSupportingDistributedTransactions()
+        {
+#if NET7_0_OR_GREATER
+            return OperatingSystem.IsWindows() && System.Runtime.InteropServices.RuntimeInformation.ProcessArchitecture != System.Runtime.InteropServices.Architecture.X86 && IsNotAzureServer();
+#elif NETFRAMEWORK
+            return System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform(System.Runtime.InteropServices.OSPlatform.Windows) && IsNotAzureServer();
```

Comment:
Nit: .NET Framework is not built/tested on non-windows, so platform check can be avoided.

Reply:
Isn't it safer to leave it as is? Happy to change it if you wish so